backend/apps/progress/views.py

```
@@ -737,8 +737,12 @@ def get(self, request):
 from .serializers import CodeSubmissionSerializer, PeerReviewSerializer
 
 
+from rest_framework.throttling import ScopedRateThrottle
+
 class CodeSubmissionView(APIView):
     permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)